gprofiler/profilers/java.py

```diff
@@ -318,8 +321,76 @@ def read_output(self) -> Optional[str]:
         ),
     ],
 )
+@dataclass
+class JvmVersion:
+    version: Version
+    build: int
+    name: str
+
+
+JAVA11_BUILD_REGEX = re.compile("\\d+")
```

Comment:
```suggestion
JAVA11_BUILD_REGEX = re.compile(r"\d+")
```

also since it's very small I suggest you just inline it? Would make the code more readable. Don't need to cache compilation, Python does it anyway :)

Reply:
>Don't need to cache compilation, Python does it anyway :)

Cool, didn't know that